### DISCUSSION TITLE
Accesibility Features

### DISCUSSION THREAD

**Discussion by radiorob** [openbroadcaster/observer#25] [Ideas]
OpenBroadcaster has many features to ensure maximum accessibility for as many users as possible.

### Themes

OpenBroadcaster Server allows for individual custom themes that support users with different visibility conditions:

- High Contrast: for users with visual impairments

- Low Contrast: for users with light sensitivity issues, such as migraines

- Dyslexic-Friendly Font: for users with Dyslexia, this theme uses a dyslexic friendly font: http://opendyslexic.org/

- Ambidextrous sidebar media and preview layout organization

- Keyboard shortcuts

**Comment by robnoordt** [comment]
I would like to bring the following to your attention. Hopefully this will provide input for the developers that will help them and help them with future updates/upgrades. A possibly underexposed issue in accessibility is the following. As a blind user I use a screen reader on my PC. For this I do not use the mouse but only navigate with the keyboard. Because of this there are things that, normally intended for mouse users, are not possible. In many radio automation programs this is where things go wrong. Often it is then the intention with dreck and drop with the mouse to put a list of mp3 files in the database. This is an action that cannot be performed with a keyboard. It would also be better if this were possible with Cut(ctrl+X( and Paste(Ctrl+V). Still waiting for a new release for the Raspberry 4, I hope this will be included in this version and the this makes it easier to select and add mp3 files, so I hope the developers can do something with this!

**Comment by radiorob** [comment]
I was tabbing around and with a combination of enter + tab it came up trying to delete everything.  We need to ensure that when deleting, the default keyboard command is to cancel the current operation.
